crates/cargo-bsp/src/project_model/build_target_mappings.rs

```diff
@@ -44,7 +45,7 @@ fn tags_and_capabilities_from_cargo_kind(
         .kind
         .iter()
         .for_each(|kind| match kind.as_str() {
-            "lib" => {
+            "lib" | "rlib" | "dylib" | "cdylib" | "staticlib" | "proc-macro" => {
```

Comment:
drobnostka jakby się wam chciało - można by najpierw zamienić kind na `BuildTargetTag` podobnie jak `metadata_kind_to_rust_extension_kind()` i jeżeli się udało zmapować to dodać go do vectora i zrobić match tylko na zmianę capabilities

Reply:
Próbowałam to zaimplementować, ale wychodziło w sumie bardziej skomplikowane niż aktualnie jest. Może by jeszcze fajnie działało, gdyby `capabilities` były zależne od`BuildTargetTag`, ale niestety nie są

Reply:
okay, no to git